docs/stressors.md

```diff
@@ -72,29 +70,33 @@ func (m *MyStressorTask) ExecAllowed(endpoint *model.Endpoint) bool { ... }
 func (m *MyStressorTask) ExecTask(endpoint *model.Endpoint, responses *MutexTaskResponses) { ... }
 ```
 
-The stressor should add a response to the task responses structure, which is located in model/api.go:
+The stressor should add a response to the task responses structure, which is located in model/api.proto:
 
 ```go
-type CPUTaskResponse struct {
-    Services []string `json:"services"`
-    Statuses []string `json:"statuses"`
+message CPUTaskResponse {
+    map<string, float> services = 1;
 }
 
-type NetworkTaskResponse struct {
-    Services []string `json:"services"`
-    Statuses []string `json:"statuses"`
-    Payload  string   `json:"payload"`
+message ServiceResponse {
+    string protocol = 1;
+    string status = 2;
 }
 
-type MyTaskResponse struct {
-    Services []string `json:"services"`
-    Statuses []string `json:"statuses"`
+message NetworkTaskResponse {
+    repeated string services = 1;
```

Comment:
Are the "= 1", "= 2", "= 3" indicating the order in which parameters should be given?

Reply:
Protocol buffers requires us to define the order that the parameters will be sent over the network. It can be given in any order in the Go code.  
https://protobuf.dev/programming-guides/proto3/#assigning